wsclient/pom.xml

```diff
@@ -46,6 +46,10 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.1</version>
+          <configuration>
+                    <source>1.6</source>
+                    <target>1.6</target>
+                </configuration>
```

Comment:
Needed for the antlr4 plugin used to parse ecl record definitions. I haven't seen any issues created by this config change; all the dsp & HIPIE developers to date have at least 1.7 on their machines, or use Eclipse versions with at least 1.6 configured for use.